udata_front/theme/gouvfr/assets/js/components/vanilla/dialog.js

```diff
@@ -0,0 +1,63 @@
+/**
+ *
+ * @param {HTMLElement} dialog
+ */
+function showDialog (dialog) {
+  if (dialog) {
+    dialog.classList.remove("fr-hidden");
+  }
+};
+
+/**
+ *
+ * @param {HTMLElement} dialog
+ */
+function closeDialog (dialog) {
+  if (dialog) {
+    dialog.classList.add("fr-hidden");
+  }
+  window.localStorage.setItem(dialog.id, "true");
+};
+
+/**
+ *
+ * @param {HTMLElement} dialog
+ * @param {number} triggerCount
+ * @param {string} startsWithString
+ */
+function triggerDialog(dialog, triggerCount = 1, startsWithString = '/') {
```

Comment:
I would increment triggerCount to 2, so that it displays on third page being visited?